### ISSUE TITLE
The testing results on RoadAnomaly21 and RoadObstacle21 are very low. 

### ISSUE THREAD

**Issue by ZelongZeng** [NazirNayal8/RbA#3]
Hi,
Thanks for your nice paper. 
I meet a problem. When I do the test for _RoadAnomaly21_ and _RoadObstacle21_ datasets, the 'aupr' metrics are very low (I used the pre-trained weights 'RbA+COCO outlier'). 

Here are the testing results: 
{'road_anomaly_21': {'aupr': 0.060153801641492935,
                     'auroc': 0.896429923771539,
                     'fpr95': 0.18668411637967985}}

{'road_obstacles': {'aupr': 0.05316241462096945,
                    'auroc': 0.9903306674564318,
                    'fpr95': 0.03634343782142339}}

Could you help me to solve the problem? Thanks.

**Comment by NazirNayal8** [comment]
Hi @ZelongZeng,

Thank you for your interest in our work. The labels for RoadAnomaly21 and RoadObstacles21 are hidden and evaluating on them the full set requires submitting the official benchmark: https://segmentmeifyoucan.com/leaderboard

However there are some validation samples with labels provided, and to be able to evaluate on them I have modified to code to allow that. so now you can run the following command to evaluate on their validation sets which have labels:

```
python evaluate_ood.py --out_path output/dummy --models_folder ckpts/ --datasets_folder /home/nazir/datasets --model_mode selective --selected_models swin_b_1dl_rba_ood_coco --dataset_mode selective --selected_datasets road_anomaly_21 road_obstacles
```

**Comment by ZelongZeng** [comment]
Hi @NazirNayal8 

Thank you very much for your prompt reply and help!
My problem has been solved perfectly. 

I will close this issue.
